Top-Level Requirements.md

```diff
@@ -2,22 +2,23 @@
 ## List of Terms
 * East **shall** be defined as the direction perpendicular and away to the face of the designated mining receptacle.
 ## Requirements
-### 1.	Executive System
-* 1.1 The executive system **shall** process mission control commands.  
-* 1.2 The executive system **shall** start and manage autonomous operation.  
-* 1.3 The executive system **shall** monitor and delegate system responsibilities and state in between mission subsystems.  
-* 1.4 The executive system **shall** support two classes of commands:  
-    * 1.4.1 The Autonomous Command: Commences the main mission phases and tracks autonomous mission state.  
-    * 1.4.2 Teleop Commands: A lightweight command processing mode which contains user commands to be fed to control systems.  
-* 1.5 The executive system **shall** track mission progress and mission time elapsed.  
-* 1.6 The executive system **shall** be able to preempt all subsystem.  
-* 1.7 The executive system **shall** return the system to a safe state upon failure.  
+### 1.  Executive System
+* 1.1 The executive system **shall** process mission control commands.
+* 1.2 The executive system **shall** monitor and delegate system responsibilities and state in between mission subsystems.
+* 1.3 The executive system **shall** support two modes of operation:
+    * 1.3.1 *Autonomous* Commences the main mission phases and tracks autonomous mission state.
```

Comment:
Content looks good, but the phrasing between 1.3.1 and 1.3.2 is inconsistent ("commences ..." vs. the more fragmented "lightweight user ...."). This isn't super important, but if we're looking for formality I think this is something to be addressed and it can help make it more obvious what the specific differences are between them.

Reply:
easy fix